ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/management-eapi-vrf-name-use_inband_mgmt_vrf.yml

```
@@ -0,0 +1,20 @@
+---
+type: l3leaf
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 10.10.10.0/24
+    vtep_loopback_ipv4_pool: 10.10.10.0/24
+    bgp_as: 65042
+  nodes:
+    - name: management-eapi-vrf-name-use_inband_mgmt_vrf
+      loopback_ipv4_address: 10.42.42.42
+      id: 12
+
+management_eapi:
+  vrfs:
+    - name: use_inband_mgmt_vrf
+      enabled: true
+
+expected_error_message: >-
+  'self.inputs.management_eapi.vrfs[name=use_inband_mgmt_vrf]' is set but this node is missing 'mgmt_ip' or 'ipv6_mgmt_ip'.
```

This should error out saying inband mgmt settings are missing.

---

Fixed!